.github/workflows/ci.yaml

```diff
@@ -348,9 +348,17 @@ jobs:
             echo -e "$(getent ahostsv4 $host | grep STREAM | cut -d' ' -f1 | head -n 1)\t$host"  | sudo tee -a /etc/hosts
           done
 
-      - name: Run tox job
-        run: sudo --preserve-env -H tox -e ${{ matrix.toxenv }} -- -n 3 --reruns 3
+      - name: Run the tests
+        run: tox -e ${{ matrix.toxenv }} -- -n auto --reruns 3 --pytest-container-log-level=debug
         env:
           CONTAINER_RUNTIME: ${{ matrix.container_runtime }}
           OS_VERSION: ${{ matrix.os_version }}
           TARGET: ${{ matrix.testing_target != '' && matrix.testing_target || 'obs' }}
+
+      - name: Run tests as root for podman
+        run: sudo --preserve-env tox -e ${{ matrix.toxenv }} -- -n auto --reruns 3 --pytest-container-log-level=debug
```

Comment:
why did you remove the "-H" parameter which is needed to set the home directory so that pip caching works?

Reply:
that was a mistake